src/Functions/FunctionsStringHashFixedString.cpp

```diff
@@ -218,6 +223,26 @@ struct ImplBLAKE3
 
 #endif
 
+#if defined USE_ETHASH && USE_ETHASH
```

Comment:
This duplication is strange.
Do not use `defined`, so the compiler will check if you included the config.